projects/plugins/jetpack/modules/calypsoify/class-jetpack-calypsoify.php

```diff
@@ -2,27 +2,34 @@
 /**
  * This is Calypso skin of the wp-admin interface that is conditionally triggered via the ?calypsoify=1 param.
  *
+ * @deprecated $$next-version$$ Use Automattic\Jetpack\Calypsoify instead.
+ *
  * @package automattic/jetpack
  */
 
 use Automattic\Jetpack\Calypsoify\Jetpack_Calypsoify as Calypsoify;
-use Automattic\Jetpack\Status;
 
 /**
  * Class Jetpack_Calypsoify
+ *
+ * @deprecated $$next-version$$
  */
-class Jetpack_Calypsoify {
+class Jetpack_Calypsoify extends Calypsoify {
 
 	/**
 	 * Singleton instance of `Jetpack_Calypsoify`.
 	 *
+	 * @deprecated $$next-version$$
+	 *
 	 * @var object
 	 */
 	public static $instance = false;
```

Comment:
If we want it to pass through to the parent class version, we need to remove it entirely from here. The fact that the entire class is deprecated IMO should be enough to suggest everything inside it is too.
```suggestion
```

(the non-static `$is_calypsoify_enabled` is ok to keep, but could be removed too if you'd rather)

Reply:
Got it.